packages/angular_devkit/build_angular/src/webpack/plugins/common-js-usage-warn-plugin.ts

```diff
@@ -46,7 +46,9 @@ export class CommonJsUsageWarnPlugin {
   apply(compiler: Compiler) {
     compiler.hooks.compilation.tap('CommonJsUsageWarnPlugin', compilation => {
       compilation.hooks.finishModules.tap('CommonJsUsageWarnPlugin', modules => {
-        for (const {dependencies, rawRequest, issuer} of modules as unknown as WebpackModule[]) {
+        for (const module of modules as unknown as WebpackModule[]) {
+          const {dependencies, rawRequest} = module;
+          const issuer = getIssuer(compilation, module);
```

Comment:
This should probably be moved, in scope of where it is needed. Ie: down above L74
We are only interested in the issuer when the dep is commonjs

Reply:
done